lib/docker2aci.go

```diff
@@ -48,41 +47,124 @@ type DockerURL struct {
 	Tag       string
 }
 
-type Compression int
+type SquashAcc struct {
```

Comment:
It stands for accumulator, because it accumulates all the layers' files, filenames and manifests. I can rename it to SquashAccumulator if that makes it clearer.